src/NCDatasets.jl

```diff
@@ -65,6 +68,8 @@ include("ncgen.jl")
 include("select.jl")
 include("precompile.jl")
 
+@implement_diskarray NCDatasets.Variable
```

Comment:
Are we aiming to move more of these to DiskArrays.jl?

Just having `Variable` means this PR doesn't allow Rasters.jl integration of gribb or netcdf.

Reply:
I'm experimenting with this on this PR: JuliaGeo/CommonDataModel.jl/pull/9.

The issue I'm currently facing is, I think, related to the growth of unlimited dimensions. As you predicted, this causes problems when CFVariable implements DiskArray.

Unfortunately, I can't show you the error stack on github since it needs the combination of this branch with JuliaGeo/CommonDataModel.jl/pull/9. Is there a way to tell github to run CI with a specific branch of a dependency?

Reply:
You can add a Project.toml and Mainfest.toml to /test if you need too, but its a bit of fiddling and maybe no better than just pasting errors in issues.

Yes that is fairly predictable. You will need to rebuild the chunking after `grow!`.